data/macos.yaml

```diff
@@ -518,6 +550,26 @@ sources:
 labels: [System]
 supported_os: [Darwin]
 ---
+name: MacOSNotificationCenter
+doc: macOS NotificationCenter database
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '/private/var/folders/[a-z][0-9]/*/0/com.apple.notificationcenter/db2/db'
+    - '/private/var/folders/[a-z][0-9]/*/0/com.apple.notificationcenter/db/db'
+    - '%%users.homedir%%/Library/Application Support/NotificationCenter/*.db'
+labels: [Users, Logs]
+supported_os: [Darwin]
+---
+name: MacOSPasswordHashes
```

Comment:
Doesn't this contain broader account information?

Reply:
Indeed, it has other general account info like home directory, default shells, home dir path, original full user name, user profile picture, etc. Shall we use one of the following

1) name: MacOSUsersAccountsAndPassowrd
or
2) name: MacOSUsersAccountsAndPassowords
or just
3) name: MacOSUsersAccounts

with the "doc" being:
doc: Users Accounts Information and hashed Password plist files

Reply:
Actually this was already in the macos.yaml file as "MacOSUserPasswordHashesPlistFiles" same artifacts. I will remove it from this PR and address it in the next one, when cleaning the "plistfile" names